### DISCUSSION TITLE
telecamere offline?! blink + sync module 2?!

### DISCUSSION THREAD

**Discussion by 50494554524F** [indomus/forum#1996] [Hardware & Firmware]
ho cominciato a studiare le telecamere per vedere se ce ne fosse qualcuna che può funzionare anche in assenza di rete internet/cloud... 

se le avete, potete togliermi qualche dubbio?! (le prenderei senza abbonamento)

- se internet manca continuerà a salvare i video sul sync module?! 
- i video salvati li posso linkare in qualche modo dentro home assistant?! o devo usare i servizi dell'integrazione blink per combinare qualcosa?!
- c'è rischio che amazon mi banni l'account se le usassi con home assistant anche non avendo abbonamento?!

grazie

ps: il 3d lo (ri)userei anche per altre telecamere, se avete da indirizzarmi o consigliarmi da dove partire fate pure...

**Comment by 50494554524F** [comment]
non ho trovato risposta, nemmeno da chi le recensisce... mistero, scrivo ad amazon, vediamo... speriamo perché son le più compatte

nel frattempo passiamo a quelle con microsd compatibili onvif... 

sempre la stessa domanda: con homeassistant si possono leggere le registrazioni sulla microsd o devo arrangiarmi da solo per intercettare il movimento e registrare?!? 

ne conescete qualcuna che è compatta come le blink?!

grazie

**Reply by 50494554524F** [reply]
mi sa che pure la microsd è inutile perché è inaccessibile all'esterno... evabene, cercherò camere compatte possibilmente onvif e via di codice... con molta molta calma allora

**Comment by depahk** [comment]
Ciao, ho visto ora il thread. Sconsiglio caldamente le Blink per qualsiasi integrazione.. semplicemente perché non sono di fatto integrabili in modo utile, non puoi accedere ai video e non puoi vederli. Io ho 2 cam + sync module solo perché regalate e infatti le uso per le cose più inutili.
Sto cercando anch'io cam serie da mettere in casa e stavo pensando a Ezviz o Reolink con NVR + Frigate come se n'è parlato su Telegram giorni fa.

**Reply by 50494554524F** [reply]
grazie per la conferma. per adesso ho cominciato a studiare e farò le prove con lo smartphone...

(con le blink si possono catturare le immagini di sicuro, prova un po' a vedere se riesci a catturarne una ogni 5s e simulare un video decente, 5s è l'intervallo minimo mi pare... facci sapere)

**Reply by depahk** [reply]
Avevo provato a giocarci qualche mese fa, puoi anche salvare i video quando c'è un movimento e salvare immagini ma è tutto castrato. 
Non vedo perché andare su queste telecamere davvero inutili la cui integrazione può essere dismessa da Amazon in qualsiasi momento invece che orientarsi verso telecamere che mettono a disposizione rtsp e/o onvif e quindi integrabili con qualsiasi cosa.
